docs/developer-docs/defi/tokens/ledger/setup/icrc1_ledger_setup.mdx

```diff
@@ -43,7 +43,7 @@ The URL for the ledger .did file is `https://raw.githubusercontent.com/dfinity/i
 #### Optional:
 If you want to make sure you have the latest ICRC-1 ledger files you can run the following script that downloads the files into your local directory:
 ``` sh
-curl -o download_latest_icrc1_ledger.sh "https://raw.githubusercontent.com/dfinity/ic/aba60ffbc46acfc8990bf4d5685c1360bd7026b9/rs/rosetta-api/scripts/download_latest_icrc1_ledger.sh"
+curl -o download_latest_icrc1_ledger.sh "https://raw.githubusercontent.com/dfinity/ic/aba60ffbc46acfc8990bf4d5685c1360bd7026b9/rs/ledger_suite/scripts/download_latest_icrc1_ledger.sh"
```

Comment:
@mbjorkqvist i think this path does not exist 🤔 the scripts to download the ledger still live under their `rosetta-api` path

https://github.com/dfinity/ic/tree/master/rs/rosetta-api/scripts

Reply:
some for the change in the icp ledger setup

Reply:
Thanks for catching this! I did a find/replace and then went through and reverted the invalid changes and updated the git commits, but this one fell through the cracks. I've reverted the change now.